assets/src/components/detours/detourModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { DiversionPage } from "./diversionPage"
+import { StartDetourProps } from "./detourDropdown"
+
+export const DetourModal = ({
+  detourInfo,
+  onClose,
+  show,
+}: {
+  detourInfo: StartDetourProps
+  onClose: () => void
+  show: boolean
+}) => {
+  return (
+    <div className={`c-detour-modal${show ? "" : " c-detour-modal__hidden"}`}>
```

Comment:
**suggestion:** this is a great use case for our `joinClasses` helper function (you'll also need to import it)

```suggestion
    <div className={joinClasses(["c-detour-modal", show ? "c-detour-modal__hidden" : null])}>
```